src/play/render/common/storyboard_sample.ts

```diff
@@ -0,0 +1,16 @@
+import { Howl } from "howler";
+import { StoryboardSample } from "osu-classes";
+
+export class PlayableStoryboardSample {
+  sound: Howl | null;
+
+  constructor(object: StoryboardSample, samples: Map<string, Howl>) {
+    this.sound = samples.get(object.filePath) ?? null;
+
+    if (!this.sound) return;
+
+    this.sound.once("end", () => this.sound!.unload());
+    this.sound.volume(object.volume / 100);
+    this.sound.play();
```

Comment:
Could you use howl's sound id's to allow the same sound to be played multiple times? You probably don't need to unload it either.

Reply:
Modern browsers don't allow audio autoplay, which can delay sample playback when switching tabs.